.gitlab-ci.yml

```diff
@@ -158,17 +158,42 @@ benchmarks:
     - cargo bench --features full_bench,hybrid
   when: manual
 
-build_python:
+build_python_linux:
   stage: build
   image:
     name: ghcr.io/pyo3/maturin
     # remove the image custom entrypoint because it is not supported by gitlab runners
     entrypoint: ['']
   script:
-    - maturin build --release --features python
+    # Build for glibc 2.17
+    - maturin build --compatibility manylinux_2_17 --release --features python
   artifacts:
     paths:
-      - target/wheels/*.whl
+      - target/wheels/*manylinux*.whl
+    expire_in: 3 mos
+
+build_python_windows:
```

Comment:
do you mind merging:
- `build_python_windows` with `build_windows`
- `build_python_osx` with `build_osx`

(to minimize docker env creation in CI)

Reply:
even `build_python_linux` could be merged into `build_x86_64` since we installed `maturin` in ci-rust image: http://gitlab.cosmian.com/core/cosmian-ci-images/-/blob/main/Dockerfile.rust

Reply:
the custom docker image (ghcr.io/pyo3/maturin) is needed to build manylinux package

Reply:
OK